src/NzbDrone.Core/Parser/LanguageParser.cs

```diff
@@ -235,99 +235,111 @@ private static List<Language> RegexLanguage(string title)
+                }
 
-            if (match.Groups["french"].Success)
-            {
-                languages.Add(Language.French);
-            }
+                if (match.Groups["greek"].Success)
+                {
+                    languages.Add(Language.Greek);
+                }
 
-            if (match.Groups["russian"].Success)
-            {
-                languages.Add(Language.Russian);
-            }
+                if (match.Groups["french"].Success)
+                {
+                    languages.Add(Language.French);
+                }
 
-            if (match.Groups["dutch"].Success)
-            {
-                languages.Add(Language.Dutch);
-            }
+                if (match.Groups["russian"].Success)
+                {
+                    languages.Add(Language.Russian);
+                }
 
-            if (match.Groups["hungarian"].Success)
-            {
-                languages.Add(Language.Hungarian);
-            }
+                if (match.Groups["dutch"].Success)
+                {
+                    languages.Add(Language.Dutch);
+                }
 
-            if (match.Groups["hebrew"].Success)
-            {
-                languages.Add(Language.Hebrew);
-            }
+                if (match.Groups["hungarian"].Success)
+                {
+                    languages.Add(Language.Hungarian);
+                }
 
-            if (match.Groups["polish"].Success)
-            {
-                languages.Add(Language.Polish);
-            }
+                if (match.Groups["hebrew"].Success)
+                {
+                    languages.Add(Language.Hebrew);
+                }
 
-            if (match.Groups["chinese"].Success)
-            {
-                languages.Add(Language.Chinese);
-            }
+                if (match.Groups["polish"].Success)
+                {
+                    languages.Add(Language.Polish);
+                }
 
-            if (match.Groups["bulgarian"].Success)
-            {
-                languages.Add(Language.Bulgarian);
-            }
+                if (match.Groups["chinese"].Success)
+                {
+                    languages.Add(Language.Chinese);
+                }
 
-            if (match.Groups["ukrainian"].Success)
-            {
-                languages.Add(Language.Ukrainian);
-            }
+                if (match.Groups["bulgarian"].Success)
+                {
+                    languages.Add(Language.Bulgarian);
+                }
 
-            if (match.Groups["spanish"].Success)
-            {
-                languages.Add(Language.Spanish);
+                if (match.Groups["ukrainian"].Success)
+                {
+                    languages.Add(Language.Ukrainian);
+                }
+
+                if (match.Groups["spanish"].Success)
+                {
+                    languages.Add(Language.Spanish);
+                }
+
+                if (match.Groups["portoguese"].Success)
```

Comment:
portuguese

Reply:
Good catch - I'll fix it tomorrow morning.

Reply:
This should be fixed now.